ganga/GangaCore/Runtime/gangadoc.py

```diff
@@ -185,7 +185,7 @@ def mygetmodule(object):
             return builtin
 
 
-def doc2(thing, title='Python Library Documentation: %s', forceload=0):
+def doc2(thing, title='Python Library Documentation: %s', forceload=0, output=None):
```

Comment:
@mesmith75 Am I missing something? I can't see this argument used in the function anywhere

Reply:
It is because the method this is replacing now has the output option. I think we always want to output with the `pager` so I ignore it, but the keyword argument still needs to be there.

Reply:
ok thanks.